### PR TITLE
Feature/isdb 350

### PR DESCRIPTION
**Pull request by Murshadhaque**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hmcts-kodatala — COMMENTED** [review]
Pls check review comments and resolve them

File: src/main/resources/request/ccdsearchrequest.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "query": {
+    "range": {
+      "last_modified": {
+        "gte": "2022-08-22",
+        "lte": "2022-08-22",
+        "boost": 2.0
+      }
+    }
+  },
+  "size":100
```
[the remark this answers is not in the document]
Comment (Murshadhaque) [reply src/main/resources/request/ccdsearchrequest.json]: Done

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -160,7 +164,7 @@ test-url: ${TEST_URL:http://localhost:4044}
 
 core_case_data:
   api:
-    url: ${CORE_CASE_DATA_API_URL:http://localhost:4452}
+    url: ${CORE_CASE_DATA_API_URL:http://ccd-data-store-api-aat.service.core-compute-aat.internal}
```
[the remark this answers is not in the document]
Comment (Murshadhaque) [reply src/main/resources/application.yaml]: Done

File: src/main/java/uk/gov/hmcts/reform/prl/filter/CafCassFilter.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import uk.gov.hmcts.reform.prl.services.cafcass.PostcodeLookupService;
+
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+@Service
+@Slf4j
+public class CafCassFilter {
+    public static final String CAFCAAS_CASE_TYPE_OF_APPLICATION_LIST_NOT_CONFIGURED = "cafcaas.caseTypeOfApplicationList not configured";
```
[the remark this answers is not in the document]
Comment (Murshadhaque) [reply src/main/java/uk/gov/hmcts/reform/prl/filter/CafCassFilter.java]: Done

File: src/main/java/uk/gov/hmcts/reform/prl/mapper/cafcass/CcdToCafcassObjectMapper.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CcdToCafcassObjectMapper {
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
```
[the remark this answers is not in the document]
Comment (Murshadhaque) [reply src/main/java/uk/gov/hmcts/reform/prl/mapper/cafcass/CcdToCafcassObjectMapper.java:12]: Done

**Review by hmcts-kodatala — APPROVED** [review]
OK

**Pull request merged**